package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>apriltag</name>
```

Comment:
Should this be `apriltags` to be consistent with the repository name?

Reply:
I named it `apriltag` according to the ICRA 2011 paper and the description on the webside:

> AprilTag is a visual fiducial system ...

What is the official name? I.e. I haven't seen the plural `apriltags` as a proper name in any of the references.

Reply:
Right, just looking at the repository name, which conventionally matches up with the ROS package name.